### PR TITLE
[HAB-93] Prettier GitHub action

### PR DESCRIPTION
**Pull request by himikheal**
Added yaml file to workflows folder in .github

### CODE REVIEW

File: src/app/(tabs)/friends.tsx
Comment (owengretzinger) [inline src/app/(tabs)/friends.tsx:1]: This code shouldn't be changed

File: src/components/HabitInvite.tsx
Comment (owengretzinger) [inline src/components/HabitInvite.tsx:1]: This code shouldn't be changed

File: .github/workflows/quality.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "npm"
+      - name: Install dependencies
+        run: npm install -D
+      - name: Check formatting with Prettier
+        run: npm run prettier:check:ci
+      - name: Format with Prettier
+        run: npm run prettier:format
```
Comment (owengretzinger) [inline .github/workflows/quality.yaml:31]: Does this also push the changes to the PR?

**Review by owengretzinger — CHANGES_REQUESTED** [review]

**Comment by owengretzinger** [comment]
Also is it failing its own check now??

File: .github/workflows/quality.yaml (hunk 1 above)
Comment (himikheal) [reply .github/workflows/quality.yaml:31]: I believe it does? It should run the prettier:format script

**Comment by owengretzinger** [comment]
https://github.com/owengretzinger/habits-together/pull/41

**Pull request closed without merging**